src/main/java/appeng/items/parts/PartType.java

```diff
@@ -263,16 +274,40 @@ String getUnlocalizedName()
 		this.integrations = Collections.unmodifiableSet( integrations );
 		this.myPart = c;
 		this.extraName = en;
-		if ( Platform.isClient() )
+
+		boolean enabled = true;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "enabled" which hides the field declared at line 262. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)